fe-nc-news-project/src/Home.jsx

```diff
@@ -1,3 +1,44 @@
+import { useEffect, useState } from "react";
+import { fetchArticles } from "./utils.js";
+import { Link } from "react-router-dom";
+
 export default function Home() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles().then(({ data }) => {
+      setArticles(data.articles);
+    });
+  }, []);
+
+  function handleClick() {
+    setArticles();
+  }
 
-}
\ No newline at end of file
+  return (
+    <>
+      <ul>
+        {articles.map(
+          ({ article_id, title, author, votes, article_img_url }) => {
+            return (
+              <Link key={article_id} to={`/articles/${article_id}`}>
```

Comment:
Although you will be asked to do this in another ticket please only focus on the stuff this ticket is asking for